### DISCUSSION TITLE
When Loading the Design I get 403 and then it loads Dashboard in designs frame

### DISCUSSION THREAD

**Discussion by izakhearn** [Budibase/budibase#6240] [Help]
Greetings all. I followed the docker compose setup. 

When I try to switch to the design section of an app in the network request and console I get an error 403 for /global/self. 

I tried to look at issues before but cant see why it would effect me at this stage as its a brand new install using the standard compose file no modifications nothing. 

![image](https://user-images.githubusercontent.com/25284121/172688978-7891e6e1-953a-40b0-b5d4-123a12fc8f12.png)

I hope someone can assist me as I want to try the platform out.

**Comment by melohagan** [comment]
Hi @izakhearn 

It may be handiest to try the CLI: https://docs.budibase.com/docs/budibase-cli-setup

Let me know if that works for you!
